google_health/tests/test_pull_api.py

```diff
@@ -18,110 +12,122 @@
     _load_cached_file,
     _write_cached_file,
 )
-
-
 class TestGoogleHealthTrends:
-    def test_class_state(self):
-
-        params = read_params()
-        ght = GoogleHealthTrends(ght_key=params["ght_key"])
-
-        res = ght.query("2020-05-05", "2020-05-06", geo_id="AL", dma=False)
-        assert [x for x in res.keys()] == ["lines"]
-        assert [x["term"] for x in res["lines"]] == [
-            "/m/0m7pl",
-            "why cant i smell or taste",
-            "loss of smell",
-            "loss of taste",
-        ]
-
-        assert len(res["lines"][0]["points"]) == 2
-        assert set([x for x in res["lines"][0]["points"][0].keys()]) == {"date", "value"}
-        assert res["lines"][0]["points"][0]["date"] == "May 05 2020"
-        assert res["lines"][0]["points"][1]["date"] == "May 06 2020"
-
-    def test_class_dma(self):
-
-        params = read_params()
-        ght = GoogleHealthTrends(ght_key=params["ght_key"])
-
-        res = ght.query("2020-05-05", "2020-05-06", geo_id=519, dma=True)
-        assert [x for x in res.keys()] == ["lines"]
-        assert [x["term"] for x in res["lines"]] == [
-            "/m/0m7pl",
-            "why cant i smell or taste",
-            "loss of smell",
-            "loss of taste",
-        ]
-
-        assert len(res["lines"][0]["points"]) == 2
-        assert set([x for x in res["lines"][0]["points"][0].keys()]) == {"date", "value"}
-        assert res["lines"][0]["points"][0]["date"] == "May 05 2020"
-        assert res["lines"][0]["points"][1]["date"] == "May 06 2020"
-
+     def test_class_state(self):
+         
+         # TODO
```

Comment:
please add the Github issue here and to the other TODOs.

Reply:
Sorry I wasn't clearer on this.  You should create a github issue called something like "Get the ght tests to work without authentication" and explain a bit more in the description.  Then, in the commented out tests you should leave a comment that says "TODO(#XXX):  reenable these tests" where XXX is the number of the issue you created.